main_routers/live2d_router.py

```
@@ -0,0 +1,1139 @@
+    except Exception as e:
+        logger.error(f"获取模型配置失败: {e}")
+        return JSONResponse(status_code=500, content={"success": False, "error": str(e)})
+
+
+@router.post('/api/live2d/model_config/{model_name}')
+async def update_model_config(model_name: str, request: Request):
+    """更新指定Live2D模型的model3.json配置"""
+    try:
+        data = await request.json()
+        
+        # 查找模型目录（可能在static或用户文档目录）
```

Ruff (0.14.8)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)